etc/ipxe/unconfigured.ipxe

```diff
@@ -0,0 +1,12 @@
+#!ipxe
+
+echo
+echo ================================================================================
+echo Warewulf v4
+echo
+echo MESSAGE: This node is unconfigured. Please have your system administrator add a
+echo          configuration for this node with HW address: {{$.Hwaddr}}
+echo
+echo Rebooting in 1 minute...
+sleep 60
```

Comment:
Will this create an infinite booting loop?  Or is there something limiting the number of boots?

Reply:
Yep, the idea is that a system admin can take that HW Address and manually configure, and the node will reboot until it is configured.

Reply:
Seems like notifying the admin in some way might be better than a reboot loop.  I'm imagining the case where there might be a bunch of nodes rebooting over and over for days. That might not be so good on the hardware.  If there was a list of these nodes and if the BMC could be configured with a temporary IP address, it might be easier for the admin to find the nodes, by doing an Ipmitool identify on them.